### ISSUE TITLE
JSON output does not include namespace

### ISSUE THREAD

**Issue by chlunde** [yannh/kubeconform#301]
If possible, it would be nice to include the namespace here to properly identify the object with the issue.
https://github.com/yannh/kubeconform/blob/master/pkg/output/json.go#L14

**Comment by yannh** [comment]
Right, but not all Kubernetes objects have a namespace, so this would be more specific for resources that do have one.. I' m trending towards a "I' d rather not implement this".. Do you have troubles finding the faulty resource in the file with the output provided?

**Comment by chlunde** [comment]
@yannh if you don't like it, we can close it. Typically only a few files change so the namespace is not _critical_.

The reason I requested it is that we're running a set of validators, like OPA and kubeconform, on a large repository. In this repository we could have 30+ objects named `db` in namespace `app-1`, `app-2` etc. 

To identify an object in kubernetes, you need namespace, name, kind and group. This is the fully qualified name in kubernetes, and we could extract this information consistently for other linters. This matches ObjectReference too. https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/object-reference/
